Causing internal clang tidy to error

drisspg opened the pull request:
Summary:
This was causing an error with clang tidy for internal version of PyTorch:
https://www.internalfb.com/diff/D47044755?dst_version_fbid=1190859734932683&transaction_fbid=1553883761684581

Test Plan: See Summary

Reviewed By: dmpolukhin

Differential Revision: D48202402

malfet reviewed (approved):
Deleted code is tested code...

drisspg committed:
Causing internal clang tidy to error (#106895)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/106895

bypass-github-pytorch-ci-checks

I initially added this file to enable c++ intelllisense for the PyTorch project internally in D47044755. However because this file is synced internally, this diff broke clang-tidy linter for the whole fbcode/caffe2 folder. So even though it added intellisense for git based version it should be reverted.

See for more context:
https://www.internalfb.com/diff/D47044755?dst_version_fbid=1190859734932683&transaction_fbid=1553883761684581

Test Plan: See Summary

Reviewed By: malfet, kit1980, dmpolukhin

Differential Revision: D48202402

fbshipit-source-id: ce17b388360914cc61af720d3afbc231fbfce1e7
